1-sum-app/sum.html

```diff
@@ -0,0 +1,25 @@
+<!Doctype html>
+    <head>
+        <title>1-sum-app</title>
+        <link rel="stylesheet" href="sum-app.css">
+        <script src="sum-app.js"></script>
+    </head>
+    <body>
+        <div class="wrapper-content">
+            <div class="input-wrapper">
+                <div>
+                 <u><h1 class="main-heading">Add Two Numbers </h1></u>
+                </div>
+                <div class="inp-row">
+                 <label>First number </label><input type="number" id="inp-text1" >
```

Comment:
Please give a meaningful name instead of `inp-text1`. Also for the other inputs.

Reply:
used instead 
. inp-first-num
. input-second-num
. output-result